testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TransactionLogAssertions.java

```diff
@@ -50,43 +46,6 @@ public static void assertTransactionLogVersion(AmazonS3 s3Client, String bucketN
         assertEquals(lastJsonEntry.get(), format("%020d.json", versionNumber));
     }
 
-    public static void assertNewVersion(AmazonS3 s3Client, String bucketName, String tableName, ThrowingRunnable runnable)
```

Comment:
nit: This one is not related to excluding the test right ?

Reply:
Right.